hamlet/command/run/sentry_release.py

```diff
@@ -30,6 +30,11 @@
     type=click.Choice(["", "react-native"]),
     help="The application framework being used",
 )
+@click.option(
+    "-d",
+    "--dist",
+    help="The distribution value",
```

Comment:
```suggestion
    help="A distribution Identifier for the file",
```

Just updating the name to align with the help for sentry-cli